R/psprintf.R

```diff
@@ -0,0 +1,36 @@
+#' Paste together sprintf-style text and variable values
+#'
+#' Helper function to sprintf a bunch of key-value (string-variableVector) pairs,
+#' then paste them together with a good separator for contructing remake recipes.
+#'
+#' @param ... list of sprintf arguments seperated by commas
+#' @param sep separator for the list of arguments, defaults to new line
+#'
+#' @export
+#' @examples
+#' \dontrun{
```

Comment:
Is the \dontrun necessary? This example runs fine for me

Reply:
Hmm. yeah I'm not sure why I had that in there. I'll remove the `\dontrun`